projects/storefrontlib/src/lib/ui/pages/pages.module.ts

```diff
@@ -40,7 +41,8 @@ const pageModules = [
   PaymentDetailsPageModule,
   ResetPasswordPageModule,
   StoreFinderPageModule,
-  ResetNewPasswordPageModule
+  ResetNewPasswordPageModule,
+  CheckoutComponentModule
```

Comment:
I added this to prove it works basically. I don't think it's a final solution. Basically the `CheckoutComponentModule` was imported in `multi-step-checkout-page-layout.module.ts` but this was deleted #1362. It kept working because it was also included in `order-confirmation-page-layout.module.ts`. Without this import the checkout wont load.

Since there isn't a checkout page, not really sure where to import this.

Reply:
The `OrderConfirmationComponent` relies on the `CheckoutService`. I assume you need to import the `CheckoutModule` into the `OrderConfirmationModule`.